tendrl/commons/jobs/__init__.py

```diff
@@ -219,6 +230,16 @@ def process_job(job):
                              }
                 )
             )
+            alert_utils.alert_job_status(
+                "finished",
+                "Job finished successfully (job_id: %s)" % job.job_id,
+                integration_id=job.payload['parameters'].get(
```

Comment:
Fetch integration id and cluster name from NS.tendrl_context, if NS.tendrl_context is not available only then look in the job.payload['parameters'] for integration_id and cluster_name

Reply:
Done